exercises/card.js

```diff
@@ -14,27 +14,45 @@ class Card{
 
 class Deck{
 	constructor(){
+		this.cards = [];
 		//takes in nothing
 		//make storage for your card objects
 	}
-	addCard( suit, faceValue ){
+	addCard(suit, faceValue){
+		var card = new Card(suit, faceValue);
```

Comment:
Here is a great example of inter-object-creation. It is important for a deck of cards to have the ability to create and store a deck of cards, so the needed functionality is achieved in this example by having the “addCard” method create and then store the cards. It is common to have objects create other objects, (as you had the bank object create accounts to be stored in the bank’s global “accounts” variable), so it is something that should be kept in mind when designing your class constructor functionality.